packages/pcr/package.json

```diff
@@ -49,10 +49,12 @@
     }
   },
   "peerDependencies": {
-    "koishi": "^4.17.1"
+    "koishi": "^4.17.1",
+    "koishi-plugin-pcr-landosol-roster": "^0.0.2"
```

Comment:
landosol-roster 应该是可选依赖才对吧，要在 peer 里吗

Reply:
在 peer 里面就变成了必须依赖了，这里不能加

Reply:
Well, in that case, you should add a `peerDependenciesMeta` instead of removing it.

```
"peerDependencies": {
  "koishi-plugin-pcr-landosol-roster": {
    "optional": true
  }
}
```

Reply:
还有这种东西